dev-site-config/navigation.config.js

```diff
@@ -0,0 +1,33 @@
+const navConfig = {
+  navigation: {
+    index: '/home',
+    links: [{
+      path: '/home',
+      text: 'Home',
+      pageTypes: ['home'],
+    }, {
+      path: '/components',
+      text: 'Components',
+      pageTypes: ['doc'],
+      capabilities: {
+        devTools: true,
+      },
+    }, {
```

Comment:
will have component's pages? or should they all be tools docs?

Reply:
They should all be tools docs, but the functional testing package only has docs right now. I left the section for now so i didn't have to move those docs in this PR.